### PR TITLE
add stopwatch

### PR DESCRIPTION
**Pull request by NVyrsta**
- [DEMO LINK](https://NVyrsta.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1 +1,16 @@
...
+btn.addEventListener('click', (event) => {
+  if (event.target.innerText === 'PAUSE') {
+    event.target.innerText = 'START';
+  } else {
+    event.target.innerText = 'PAUSE';
+  }
+
+  secondHand.classList.toggle('play-animation');
+  msHand.classList.toggle('play-animation');
+});
```
Comment (OlhaYevstifieieva) [inline src/main.js:16]: This solution is ingenious!

**Review by OlhaYevstifieieva — APPROVED** [review]
Perfect job! ✨ I love it 🙌🏻

File: src/main.js (hunk 1 above)
Comment (NVyrsta) [reply src/main.js:16]: Thanks:)

**Review by Denchepornyuk — APPROVED** [review]
Screenshot 2022-12-13 at 16 34 42

a great initiative with button 🔥
Probably text should be changed, when the time is running should be `pause`

**Pull request closed without merging**